Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,448 @@
+import {Grid, GridItem, Box, Flex, Spacer, Text, Center, useColorMode, VStack, Image, SimpleGrid} from "@chakra-ui/react";
+import HeadingCell from "../Shared/HeadingCell";
+import CONSTANTS from "../../data-constants.json"
+import getColor from "../Shared/colors";
+import images from "../Motor_Motion/Icons/Images"
+import RangeCell from "../Shared/RangeCell";
+
+export default function Motor_Motion(props) {
+    const { colorMode } = useColorMode();
+
+    const borderCol = getColor("border", colorMode);
+    const greenBgCol = getColor("greenBg", colorMode);
+    const greenTxtCol = getColor("txtGreenBg", colorMode);
+    const redBgCol = getColor("redBg", colorMode);
+    const redTxtCol = getColor("txtRedBg", colorMode);
+    const headerBg = getColor("header", colorMode);
+
+    const fitType = "scale-down";
+
+    const Images = images[`${colorMode}`];
+    
+    return (
+        <Flex flex='auto' direction='column'>
+            <HeadingCell fontSize='2.2vh' label='Motor / Motion'/>
+            <Grid 
+                flex='2'
+                templateRows="1fr 1fr 1fr 1.4fr 1.4fr" 
+                templateColumns="1fr 1fr 4fr 2fr"
+                h='25%'
+                pt='1'
+                pb='1'
+            >
+
+                <GridItem
+                    rowStart={1}
+                    rowSpan={2}
+                    colStart={1}
+                >
+                    <VStack spacing='0px'>
+                        <Text fontSize='1.20vh'>F/R</Text>
+                        <Center h='2.6vh' w='2.6vh' borderWidth='2px' borderRadius='md' textAlign='center'>
+                            <Text as='b' fontSize='1.6vh'>{props.data?.fr_out[0] ? 'T' : 'F'}</Text>
+                        </Center>
+                    </VStack>
+                </GridItem>
+                <GridItem
+                    rowStart={1}
+                    rowSpan={2}
+                    colStart={2}
+                >
+                    <VStack spacing='0px'>
+                        <Text fontSize='1.20vh'>State</Text>
+                        <Center h='2.6vh' w='2.6vh' borderWidth='2px' borderRadius='md' textAlign='center'>
```

Comment:
```suggestion
                        <Center
                            h='3vh'
                            w='3vh'
                            borderWidth='2px'
                            borderRadius='md'
                            textAlign='center'
                            borderColor={borderCol}
                        >
```
Again, looks good either way, so it's up to you, but thoughts on using `borderCol` vs leaving the border with its default color?